buildSrc/src/test/java/io/bitrise/trace/internal/ChangeLogHelperTest.java

```diff
@@ -1,153 +1,163 @@
-        final Ref lastTag = gitHelper.getLastTag(git);
-        final List<RevCommit> newCommits = gitHelper.getNewCommits(git, lastTag);
-
-        final List<RevCommit> actual = gitHelper.filterRelevantCommits(git, newCommits, dummyModule1);
-        final RevCommit[] expected = new RevCommit[]{newCommits.get(0), newCommits.get(1)};
-        assertThat(actual, containsInAnyOrder(expected));
-    }
+  @Test
+  public void formatCommitToChangeLogEntry_ShouldHaveExpectedFormat() {
+    final String actual =
+        changeLogHelper.formatCommitToChangeLogEntry(dummyCommitMessage1).toString();
+    final String expected =
+        "* " + dummyCommitType1 + ": **" + dummyCommitTitle1 + ":** " + dummyCommitDetails1;
+
+    assertThat(actual, is(expected));
+  }
+
+  @Test
+  public void formatCommitToChangeLogEntry_ShouldBeCaseInSensitive() {
+    final String actual =
+        changeLogHelper.formatCommitToChangeLogEntry(dummyCommitMessage1.toUpperCase())
+                       .toString();
+    final String expected =
+        "* " + dummyCommitType1 + ": **" + dummyCommitTitle1.toUpperCase() + ":** "
+            + dummyCommitDetails1.toUpperCase();
+
+    assertThat(actual, is(expected));
+  }
+
+  @Test
+  public void getChangeLogEntries_ShouldFilterNotAllowedCommits() throws IOException {
+    final List<RevCommit> newCommits = gitHelper.getNewCommits(git, gitHelper.getLastTag(git));
+    final List<ChangeLogEntry> changeLogEntries =
+        changeLogHelper.getChangeLogEntries(newCommits);
+
+    final List<String> actual = changeLogEntries.stream().map(ChangeLogEntry::toString).collect(
+        Collectors.toList());
+    final List<String> expected = new ArrayList<>();
+    expected.add(changeLogHelper.formatCommitToChangeLogEntry(dummyCommitMessage4).toString());
+    expected.add(changeLogHelper.formatCommitToChangeLogEntry(dummyCommitMessage3).toString());
+
+    assertThat(actual, is(expected));
+  }
+
+  @Test
+  public void getReleaseName_ShouldReturnReleaseName() throws IOException {
+    final Ref lastTag = gitHelper.getLastTag(git);
+    final List<RevCommit> newCommits = gitHelper.getNewCommits(git, gitHelper.getLastTag(git));
+    final List<ChangeLogEntry> changeLogEntries =
+        changeLogHelper.getChangeLogEntries(newCommits);
+
+    final String actual = changeLogHelper.getReleaseName(lastTag, changeLogEntries);
+    assertTrue(actual.startsWith("### module2 - 1.0.0 - "));
+  }
+
+  @Test
+  public void getReleaseName_ChoreRelease() throws IOException {
+    final Ref lastTag = gitHelper.getLastTag(git);
+    final List<ChangeLogEntry> changeLogEntries = Collections.emptyList();
+
+    final String actual = changeLogHelper.getReleaseName(lastTag, changeLogEntries);
+    assertTrue(actual.startsWith("### module2 - 0.1.1 -"));
+  }
+
+  @Test
+  public void getNewVersion_patchVersionIncrease() {
+    final String actual =
+        changeLogHelper.getNewVersion(dummyVersionName1, ChangeLogHelper.getPatchCommitTypes());
```

Comment:
this maybe isn't the most optimal way to split the line? I presume it was auto generated and just picked the first place to line break?

Reply:
If I remove the line break, and do the auto format, the result is the same.